src/mkindex_options.hpp

```diff
@@ -312,75 +221,58 @@ parseCommandLine(LambdaIndexerOptions & options, int argc, char const ** argv)
             options.alphReduction = 0;
         }
 
-        getOptionValue(buf, parser, "genetic-code");
-        switch (buf)
+        switch (geneticCodeTmp)
         {
             case 1: case 2: case 3: case 4: case 5: case 6:
             case 9: case 10: case 11: case 12: case 13: case 14: case 15: case 16:
             case 21: case 22: case 23: case 24 : case 25:
-                options.geneticCode = static_cast<GeneticCodeSpec>(buf);
+                options.geneticCode = static_cast<GeneticCodeSpec>(geneticCodeTmp);
```

Comment:
you could actually make a valid_values-validator for this since we have them in seqan3 now!